about.html

```diff
@@ -0,0 +1,108 @@
+                     <li><a href="#">Sponsor</a></li>
+                     <li><a href="#">News</a></li>
+                     <li class="get-ticket"><a href="#">BB Consert</a></li>
+                 </ul>
+              </div>
+             </div>
+            <div class="head">
+             <div class="top-bar">
+                <div class="menu-icon">
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                </div>
+            </div>
+            <h1 class="heading">BATTLE OF THE BANDS TOUR CONCERT 2022</h1>
+            <div class="heading-description">
+                <h4>Explore our school band festivals and tours for concert and jazz bands, wind ensembles, and percussion and marching bands. From clinics and workshops to adjudicated festival competitions to custom concert tours. </h4>
+            </div>
+            <div class="reach-us">
+                <h3>For more information about the battles of the bands contact us through our Email</h3>
+                <h4>battleofbands@gmail.com</h4>
+            </div>
+        </div>
+        </section>
+        <section id="logo">
+            <div class="logo-info">
+                <h3>Battle OF Bands Concerts 2015 Logo</h3>
+                <img src="./assets/Vector.png" alt=""/>
+                <h4>Explore the roots of Africa and experience some of the nation’s finest music in the historic city of Nairobi</h4>
+            </div>
+            <div class="logo-holder">
+                <img src="./assets/bandlogo2.png" alt=""/>
+            </div>
+        </section>
+        <section id="past-battle">
+            <div class="past">
+                <h3>See the past Battle OF Bands Concerts</h3>
+                <img src="./assets/Vector.png" alt=""/>
+                <h4>Take a look ata the last two past battle of the bands concerts in Nairobi Kenya</h4>
+            </div>
+            <div class="past-consert">
+                <div class="past-holder1">
+                    <div class="overlay">
+                        <h3>2019</h3>
+                        <h4>Battle Of Band In Nairobi</h4>
+                    </div>
+                </div>
+                 <div class="past-holder2">
+                   <div class="overlay">
+                        <h3>2021</h3>
+                        <h4>Battle Of Band In Nairobi</h4>
+                    </div>
+                </div>   
+            </div>      
+        </section>
+        <section id="partners">
+            <div class="patner">
+                <img src="./assets/kbc.png" alt="kenya broadcasting"/>
+                <img src="./assets/kaka2.png" alt="kaka empire"/>
+                <img src="./assets/magik.png" alt="magic kenya"/>
+                <img src="./assets/mcsk2.png" alt="MCSK Kenya"/>
+                <img src="./assets/youth2.png" alt="youth kenya"/>
+            </div>
```

Comment:
- [ ] Kindly align these items to the center as it is on your home page to give your project uniformity
     👇 
          
![image](https://user-images.githubusercontent.com/66097371/163470287-17211ea1-cb1b-4575-a24a-141f9b779723.png)